insights/parsers/ceph_log.py

```diff
@@ -0,0 +1,39 @@
+"""
+CephLog - file ``/var/log/ceph/ceph.log``
+=========================================
+"""
+
+from .. import LogFileOutput, parser
```

Comment:
Please avoid using relative imports.

Reply:
@psachin 
Acknowledged. Changes done in latest commit.